### PR TITLE
SAK-49533 T&Q: Ability to reshuffle all the parts using random questi…

### PR DESCRIPTION
**Pull request by mpellicer**
…on pools

### CODE REVIEW

File: config/configuration/bundles/src/bundle/org/sakaiproject/config/bundle/default.sakai.properties [hunk 1]
```diff
@@ -3736,6 +3736,10 @@
 # DEFAULT: false
 # samigo.settings.backgroundcolor.enabled=true
 
+# Ability to reshuffle all the parts using random questions from question pools
+# DEFAULT: false
+# amigo.all.random.parts.reshuffle.enabled=true
```
Comment (jesusmmp) [inline config/configuration/bundles/src/bundle/org/sakaiproject/config/bundle/default.sakai.properties:3741]: First letter s is missing:

**s**amigo.all.random.parts.reshuffle.enabled=true

Comment (mpellicer) [reply config/configuration/bundles/src/bundle/org/sakaiproject/config/bundle/default.sakai.properties:3741]: I meant amigo, gracias amigo!

Comment (mpellicer) [inline config/configuration/bundles/src/bundle/org/sakaiproject/config/bundle/default.sakai.properties:3741]:
```suggestion
# samigo.all.random.parts.reshuffle.enabled=true
```

**Review by jesusmmp — APPROVED** [review]
Awesome!

**Pull request merged**